docs/src/_includes/components/docs-toc.html

```diff
@@ -1,5 +1,5 @@
 <nav class="docs-toc c-toc" aria-labelledby="js-toc-label">
-    {%- if all_content | toc | safe -%}
+    {%- if content | toc | safe -%}
```

Comment:
```suggestion
    {%- if all_content | toc | safe -%}
```

I think this was accidentally overwritten. It looks confusing to check if `content` alone has toc and then show toc for `all_content`.